src/main/webapp/app/entities/participation-submission/participation-submission.component.html

```diff
@@ -0,0 +1,50 @@
+<h1>Submissions for Participation with id {{ participationId }}</h1>
```

Comment:
Please show the name of the student here, which would be `participation.student.name`
(See my comment below that firstName might not be enough).

I guess for this you need to access to the participation. Easiest way to do this, would be to load it based on the participationId.

Reply:
I additionally added the exercise title:

<img width="1130" alt="image" src="https://user-images.githubusercontent.com/41108487/63980915-46627c80-cabd-11e9-81a0-273c42bc469b.png">